Tools/LyTestTools/ly_test_tools/environment/watchdog.py

```diff
@@ -138,8 +138,8 @@ def __init__(self, process_id, interval=1, raise_on_condition=True, name='proces
         :param unresponsive_timeout_seconds: How long the process needs to be unresponsive for in order for the watchdog
         to report (in seconds).
         """
-        if not WINDOWS:
-            raise (NotImplementedError, "Process watchdog is only implemented on Windows.")
+        if not ly_test_tools.WINDOWS:
+            pass # TODO add non-windows support
```

Comment:
Are we still supposed to link to issues for to-dos?

Reply:
I do not plan on linking an internally-tracked issue, and we were blocked from moving these tasks into GHI. The issue is the immediate follow up task.